sound/soc/sof/intel/cnl.c

```diff
@@ -41,6 +41,7 @@ irqreturn_t cnl_ipc4_irq_thread(int irq, void *context)
 	u32 hipcida, hipctdr;
 
 	hipcida = snd_sof_dsp_read(sdev, HDA_DSP_BAR, CNL_DSP_REG_HIPCIDA);
+	hipctdr = snd_sof_dsp_read(sdev, HDA_DSP_BAR, CNL_DSP_REG_HIPCTDR);
```

Comment:
you may want to mention that the IPC3 code already reads those registers upfront.
I don't think we need a Fixes tag since this is still WIP and not used by anyone outside of Intel.

Reply:
OK, just a sec and push an update for the commit message!